docs/details/dashboards/dashboard-inventory.md

```diff
@@ -42,6 +67,8 @@ By expanding the entry from the options column, you can check the node labels an
 
 Starting with PMM 2.38.0, you can see the number of agents running on any particular node. When you click on any node, the UI navigates to the view of agents, which is filtered to display only agents related to that specific node. 
 
+Also, starting with PMM 2.40.0, when you click on the link in the **Services** column, you can see the services that are running on that specific node.
```

Comment:
You will only see that one particular service (which the name belongs to) not all of the services running on that node.

Reply:
Done.